be/src/io/cache/block/block_file_cache.cpp

```diff
@@ -32,6 +32,9 @@ namespace fs = std::filesystem;
 namespace doris {
 namespace io {
 
+const std::string IFileCache::FILE_CACHE_VERSION = "2.0";
+const int IFileCache::KEY_PREFIX_LENGTH = 3;
```

Comment:
`const` -> `constexpr` is better

Reply:
string can't use constexpr, I'll use constexpr for `KEY_PREFIX_LENGTH` in next PR.

Reply:
`std::string_view` can be constexpr[Lol]